src/en/introducing-2.md

```diff
@@ -35,8 +35,7 @@ cloud you only need one controller, but can drive several models.
  - **Providers:** As well as updates to the Azure provider and 
 the new Rackspace provider, Juju now makes extensive use of LXD. This makes 'local' 
 models leaner and faster and also brings benefits to placing containers within 
-models. You can find out more about setting up 
-[LXD for Juju here][lxd].
+models. You can find out more about [Using LXD with Juju][clouds-lxd].
```

Comment:
The literal meaning seems broken without the link, eg. You can find out more about[Using LXD with Juju (where?).

Reply:
:astonished: